src/components/GoalsCover.js

```diff
@@ -62,8 +62,8 @@ class GoalsCover extends React.Component {
   }
 
   componentDidMount() {
-    const state = this.populateGoals();
-    this.setState(state);
+    this.populateGoals();
+    window.onresize = () => this.populateGoals();
```

Comment:
```suggestion
    window.addEventListener('resize', this.populateGoals);
```

Reply:
Also, maybe you want to throttle that:

```suggestion
    window.addEventListener('resize', throttle(this.populateGoals.bind(this), 400));
```

Reply:
good idea. Fix coming (need to add `throttle` from `lodash`)